### PR TITLE
NON-248: Non-Associations Link

### PR DESCRIPTION
**Pull request by ashleygyngell**
Adding "Non-associations" link:

- Component already exists but is missing the link to a prisoners non associations.
- Feature flag added that displays/doesn't display the link depending on the active caseload.

### CODE REVIEW

**Review by Mjwillis — APPROVED** [review]
LGTM

File: server/utils/utils.ts [hunk 1]
```diff
@@ -468,6 +468,11 @@ export const neurodiversityEnabled = (agencyId: string): boolean => {
   return isEnabled
 }
 
+export const nonAssociationsEnabled = (agencyId: string): boolean => {
+  const isEnabled = config.nonAssociationsPrisons?.includes(agencyId)
+  return isEnabled
```
Comment (petergphillips) [inline server/utils/utils.ts:473]: pointless variable

Comment (xoen) [reply server/utils/utils.ts]: Removed [here](https://github.com/ministryofjustice/hmpps-prisoner-profile/pull/298/commits/c76d6d3b23619ee26aeacc4f50c7db982901b6e2) 👍

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**